.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# Changes
+
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+<!-- Describe your changes here- ideally you can get that description straight from
+your descriptive commit message(s)! -->
+
```

Comment:
Should we also encourage contributors to link their PR to a GitHub issue?

Example:

```
<!-- If this PR fixes a GitHub issue, please mention it like so:

Fixes #<insert issue number here>

-->
```

Reply:
I like that.  Will add.

Reply:
+1